src/ChatFeed/MessageForm/Attachment/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+import { Props } from './props';
+import { styles } from './styles';
+
+export const Attachment: React.FC<Props> = ({
+  fileName,
+  imageUrl,
+  onRemove = () => {},
+  style = {},
+}: Props) => {
+  const [hovered, setHovered] = useState(false);
+  const hoverStyle = { opacity: hovered ? '1' : '0' };
+
+  return (
+    <span
+      className="ce-message-attachment-preview"
+      onMouseEnter={() => setHovered(true)}
+      onMouseLeave={() => setHovered(false)}
+      style={{ ...styles.attachmentWrapper, ...style.attachmentWrapper }}
```

Comment:
maybe we call the overriding style object something like `customStyles`

I don't think it's immediately clear what is going on when there is the `{...styles, ...style}` pattern,
 so it would be `{...styles, ...customStyles}`

Reply:
Actually fair point. My initial assumption was that `style` is just native but it's not the same actually.

The style props takes properties, but this is taking an interface that we make up.

I like you're idea, I'll make an Issue about it and one of us can do it sometime 👍